packages/@aws-cdk/aws-lambda-event-sources/README.md

```diff
@@ -0,0 +1,106 @@
+## AWS Lambda Event Sources
+
+This module includes classes that allow using various AWS services as event
+sources for AWS Lambda via the high-level `lambda.addEventSource(source)` API.
+
+NOTE: In most cases, it is also possible to use the resource APIs to invoke an
+AWS Lambda function. This library provides a uniform API for all Lambda event
+sources regardless of the underlying mechanism they use.
+
+### SQS
+
+Amazon Simple Queue Service (Amazon SQS) allows you to build asynchronous
+workflows. For more information about Amazon SQS, see Amazon Simple Queue
+Service. You can configure AWS Lambda to poll for these messages as they arrive
+and then pass the event to a Lambda function invocation. To view a sample event,
+see [Amazon SQS Event](https://docs.aws.amazon.com/lambda/latest/dg/eventsources.html#eventsources-sqs).
+
+To set up Amazon Simple Queue Service as an event source for AWS Lambda, you
+first create or update an Amazon SQS queue and select custom values for the
+queue parameters. The following parameters will impact Amazon SQS's polling
+behavior:
+
+* __VisibilityTimeout__: May impact the period between retries.
+* __TimeToWait__: Will determine long poll duration. The default value is 20 seconds.
+
+```ts
+import { SqsEventSource } from '@aws-cdk/aws-lambda-event-sources';
+
+const queue = new sqs.Queue(...);
+
+lambda.addEventSource(new SqsEventSource(queue, {
+  batchSize: 10 // optional
```

Comment:
You list the "parameters that impact the polling" in the previous paragraph, and then you demo one that's not in the list. That's confusing.

Reply:
It's stolen from the Lambda documentation. I will make it cleaer.